packages/mui-material/src/CardHeader/CardHeader.js

```diff
@@ -35,6 +35,9 @@ const CardHeaderRoot = styled('div', {
   display: 'flex',
   alignItems: 'center',
   padding: 16,
+  [`& > .${typographyClasses.root}`]: {
```

Comment:
I use this to specifically target the a [direct child of Typography](https://github.com/mui/material-ui/blob/next/packages/mui-material/src/CardHeader/CardHeader.js#L95-L104) used in this component.